201-vmss-automatic-repairs-slb-health-probe/azuredeploy.parameters.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
remove all non-GEN values and use the defaultValues you have defined in the template